charts/tidb-cluster/templates/monitor-job.yaml

```diff
@@ -4,18 +4,18 @@ kind: Job
 metadata:
   name: {{ .Values.clusterName }}-monitor-configurator
   labels:
-    app: {{ template "tidb-cluster.name" . }}
-    component: monitor-configurator
-    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "tidb-cluster.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Values.clusterName }}
+    app.kubernetes.io/component: monitor-configurator
```

Comment:
this is just a job, not need to define a new component for it, ```monitor``` will be better

Reply:
If the labels are the same with monitor deployment, then monitor service would select it which doesn't make sense.